File: Assignment-34/Program3.java
import java.util.*;

class Program3
{
      public static void main(String arg[])
      {
            Scanner sobj = new Scanner(System.in);

            System.out.println("Enter The Size of Elements That You Want To Create:");
            int iSize = sobj.nextInt();

            int Arr[] = new int[iSize];
            
            Number obj = new Number();
            obj.Accept(Arr);
            
            System.out.println("Enter The Frequency Of elements");
            int iNo = sobj.nextInt();
           int iRet =  obj.LastOCC(Arr,iNo);
          System.out.print(iRet+"\t");
           
      }
}
class Number
{
      
      public void Accept(int Arr[])
      {
            Scanner sobj = new Scanner(System.in);
            System.out.println("Enter The Elements:");
            for(int icnt = 0;icnt<Arr.length;icnt++)
            {
                  Arr[icnt] = sobj.nextInt();
            }
      }
      public int LastOCC(int Arr[],int iNo)
      {
            int icnt = 0;
            for(int i = 0;i<Arr.length;i++)
            {
                  if(Arr[i]==iNo)
                  {
                        icnt = i;
                  }
            }
            return icnt;
      }

}

File: Assignment-56/Program3.c
#include<stdio.h>

int Small(char *str)
{
     static int iCnt = 0;

     if(*str !='\0')
     {
            if(*str >='a' && *str <= 'z')
            {
                  iCnt++;
            }

            str++;

            Small(str);
     }
     return iCnt;
}
int main()
{
      int iRet = 0;
      char Arr[20];

      printf("Enter the string : \n");
      scanf("%[^'\n]s",Arr);

      iRet = Small(Arr);

      printf("%d\n",iRet);

      return 0;
}